src/main/java/blusunrize/immersiveengineering/common/blocks/metal/conveyors/DropConveyor.java

```diff
@@ -84,14 +89,17 @@ else if(contact&&isEmptySpace(getTile().getWorld(), posDown, inventoryTile))
 
 	boolean isEmptySpace(World world, BlockPos pos, TileEntity tile)
 	{
-		if(world.isAirBlock(pos))
-			return true;
-		if(tile instanceof IConveyorTile)
-			return true;
 		BlockState state = world.getBlockState(pos);
-		if(state.getBlock() instanceof TrapDoorBlock)
-			return state.get(TrapDoorBlock.OPEN);
-		return false;
+		VoxelShape shape = state.getCollisionShape(world, pos);
+		// Combining voxelshapes is a little expensive, so only calculate
+		// when the voxelshape changes. Identity compare is sufficent since they're
+		// usually precomputed.
+		if (shape != cachedDownShape)
+		{
+			cachedOpenBelow = VoxelShapes.compare(REQ_SPACE, shape, IBooleanFunction.AND);
```

Comment:
I think you're missing a `!` here, this gives `false` for `shape = VoxelShapes.empty()`.

Reply:
Indeed.